README.md

```diff
@@ -600,22 +611,21 @@ Image(resource: \.home_black_18)
 
 ### Example 8 - pass font
 
-Fonts resources directory is `commonMain/resources/MR/fonts`.  
-Font name should be this pattern: `<fontFamily>-<fontStyle>` like:
+Fonts resources directory is `commonMain/moko-resources/fonts`.
+Supported type of resources:
+- `ttf`
+- `otf`
 
+If we add to `commonMain/moko-resources/fonts` files:
 - `Raleway-Bold.ttf`
-- `Raleway-Regular.ttf`
+- `Raleway-Regular.otf`
 - `Raleway-Italic.ttf`
-  Supports `ttf` and `otf` resources.
 
-If we add to `commonMain/resources/MR/fonts` files:
-
-- `Raleway-Bold.ttf`
-- `Raleway-Regular.ttf`
-- `Raleway-Italic.ttf`
+We got autogenerated:
+- `MR.fonts.raleway_italic_ttf`,
+- `MR.fonts.raleway_regular_otf`,
+- `MR.fonts.raleway_bold_italic_ttf`
```

Comment:
extension not should be generated

Reply:
mistake when write readme, generated property without extension

Reply:
lets fix readme:)

Reply:
sorry, lose this change, pushed